kubernetes/helm/startree-thirdeye/templates/coordinator/deployment.yaml

```diff
@@ -107,6 +107,11 @@ spec:
               -Djavax.net.ssl.keyStore=/opt/thirdeye/server/keystore/keystore
               -Djavax.net.ssl.keyStorePassword=changeit
               {{- end }}
+              {{- $cpuValue := .Values.coordinator.resources.requests.cpu -}}
```

Comment:
I think we should try to put more complexity in the helper for the moment. 

can we have a helper `lessOrEqualTo1Cpu` (or smthing like that) that returns true if cpuValue is `le` to 1000.

ie we don't have to write the test 
```
{{ if le $normalizedCpu 1000.0 }}
```
in all charts. This test requires an understanding of the helper method.

it can be hidden in a helper function

we could have
```
{{if include "lessOrEqualTo1Cpu" .Values.coordinator.resources.requests.cpu }}
-XX:ActiveProcessorCount=2
{{- end }}
```

--> note: I may be totally wrong here - not sure how types and imports work in helm

Reply:
nevermind it does not bring much value and may be complex to implement

Reply:
it's done